lib/buffer.js

```diff
@@ -1026,7 +1044,7 @@ Buffer.prototype.lastIndexOf = function lastIndexOf(val, byteOffset, encoding) {
 };
 
 Buffer.prototype.includes = function includes(val, byteOffset, encoding) {
-  return this.indexOf(val, byteOffset, encoding) !== -1;
+  return bidirectionalIndexOf(this, val, byteOffset, encoding) !== -1;
```

Comment:
```suggestion
  return bidirectionalIndexOf(this, val, byteOffset, encoding, true) !== -1;
```

This is a test failure that GHA mentions that seemed fairly easy to identify, I'm not sure if the others are related